tensorflow/core/kernels/reverse_sequence_op.cc

```diff
@@ -174,7 +174,7 @@ class ReverseSequenceOp : public OpKernel {
   REGISTER_REVERSE_SEQUENCE(type, int32);      \
   REGISTER_REVERSE_SEQUENCE(type, int64);
 
-TF_CALL_NUMBER_TYPES(REGISTER_REVERSE_SEQUENCE_LEN);
+TF_CALL_POD_TYPES(REGISTER_REVERSE_SEQUENCE_LEN);
```

Comment:
just add
```
TF_CALL_bool(REGISTER_REVERSE_SEQUENCE_LEN)
```

here
etc.

there are other GPU pod types that you're not including in your declaration in register_types; i recommend reverting that change.